include/llbuild/Basic/LeanWindows.h

```diff
@@ -0,0 +1,25 @@
+//===- LeanWindows.h --------------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef LLBUILD_BASIC_SHELLUTILITY_H
+#define LLBUILD_BASIC_SHELLUTILITY_H
+
+#if !defined(_WIN32)
+#error "LeanWindows.h should only be included on Windows."
+#endif
+
+#define NOMINMAX
```

Comment:
Doesn't seem like it, as this wonderfully worded article complains about - http://lolengine.net/blog/2011/3/4/fuck-you-microsoft-near-far-macros